### PR TITLE
fix #76511 kiiw.icu

### LINKED ISSUE #76511: kiiw.icu
### Issue URL (Anti Adblock Script)
 [https://kiiw.icu/y9mG](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fkiiw.icu%2Fy9mG)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Edge
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by wout-codes**
#76511

### CODE REVIEW

File: AnnoyancesFilter/sections/tweaks.txt [hunk 1]
```diff
@@ -646,7 +646,7 @@ filerio.in#%#//scriptlet("adjust-setTimeout", "tick()", "", "0.02")
...
...imagenesderopaparaperros.com,c-ut.com,toroox.com,saungfirmware.id,shrinkme.in,fir3.net,xz2.xyz,softairbay.com,link1s.net,cashurl.in,doctor-groups.com,bitcoinly.in,clk.ink,abdeo8.com,apksvip.com,gibit.xyz,claimcrypto.cc,shortenbuddy.com,btcdot.xyz,zlink.xyz,pkr.pw,shrinkbtc.cc,todaynewspk.win,manikusa.com,cut.my.id,linkmit.us,try2link.net,stfly.me,dz4win.com,real-sky.com,bolssc.com,short2.cash,fx4vip.com,fcdot.lc,cutdl.xyz,shrinkurl.org,mediumarticles.com,asupload.com,exee.io,srt.leechpremium.link,adsrt.live,shorter.fun,cheappath.com,fcc.lc,shorthitz.com,savelink.site,linkshorten.xyz,sk-ip.tech,tmearn.com,pozvi.icu,samapro.me,adsy.pw,owllink.net,mondainai.moe,2ota.com,popimed.com,cut4fun.com,aii.sh,sekilastekno.com,miuiku.com,articlix.com,intothelink.com,newstomat.com,pingit.link,slink.bid,7r6.com,loptelink.vip,iir.ai,biroads.com,win10.vn,rozfly.com,mitly.us,adsrt.net,afly.us,bitconews.net,tii.ai,linkviet.xyz,coredp.com,linkrex.net,bit-url.com,adsrt.org,bestearnonline.com,dutchycorp.ovh,gamesrs.com,shorten.sh,shotly.io,ouofly.com,clicksbee.com,shorterall.com,dutchycorp.space,linkshrnk.com,linkad.in,fc.lc,adslinkfly.online,shrinkme.io#%#//scriptlet("set-constant", "blurred", "false")
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/tweaks.txt:649]: what this rule does?

Comment (wout-codes) [reply AnnoyancesFilter/sections/tweaks.txt:649]: Yeah, why did I add it there. Lol.

**Review by Alex-302 — APPROVED** [review]

File: AnnoyancesFilter/sections/tweaks.txt (hunk 1 above)
Comment (AdamWr) [reply AnnoyancesFilter/sections/tweaks.txt]: It causes that timer doesn't stop if you go to another tab when countdown is counting.

Comment (wout-codes) [reply AnnoyancesFilter/sections/tweaks.txt]: Oh okay. I didn't find any errors.

**Pull request merged**